externs/shaka/player.js

```diff
@@ -885,6 +885,7 @@ shaka.extern.HlsManifestConfiguration;
  *   disableThumbnails: boolean,
  *   defaultPresentationDelay: number,
  *   segmentRelativeVttTiming: boolean,
+ *   sequenceMode: boolean,
```

Comment:
I wonder if it would make sense to have this be on the `DashManifestConfiguration` and the `HlsManifestConfiguration`, instead of just the `ManifestConfiguration`.
That would allow users to optionally go back to the previous status quo, of having sequence mode on HLS but not DASH, if they desire that.
It would also help us better convey that, currently, HLS is sequence-mode-only. Since we could hold off on adding the HLS config until HLS supports non-sequence-mode.

Reply:
It makes sense to me, I'm going to change it